go/README.md

```diff
@@ -0,0 +1,19 @@
+# Go lambda function boilerplate
+
+## Install dependencies
+The compressed function file will need all its dependencies so all install them in a folder called package.
```

Comment:
`The compressed function file will need all its dependencies installed in a folder called package.`